itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCompactor.java

```diff
@@ -2314,6 +2315,10 @@ public void testCompactionOnDataLoadedInPath() throws Exception {
 
     executeStatementOnDriver("load data inpath '" + path002.toString() + "' into table comp3", driver);
     executeStatementOnDriver("load data inpath '" + path002.getParent().toString() + "' into table comp3", driver);
+    Table table3 = hmsClient.getTable("default", "comp3");
+    Path delta2bucket0  = fs.globStatus(new Path(table3.getSd().getLocation(), "delta_0000002_0000002_0000/000000_0"))[0].getPath();
+    Path emptyBucket = new Path(delta2bucket0.toString().replace("000000_0", "000003_0"));
+    fs.create(emptyBucket, true);
```

Comment:
Who would create an empty file in an ACID table?
It is a very important precondition that nobody should mess around the files under an ACID table directory.